### ISSUE TITLE
DAISY Pipeline has unusual behavior in Windows compared to other applications

### ISSUE THREAD

**Issue by GeorgeKerscher** [daisy/pipeline-ui#58]
When I use an application, I normally launch it and then will close it with ALT + F4. However, when you use ALT + F4, the DAISY Pipeline will be minimized to the task bar. It no longer will  appear in the ALT + Tab order to cycle between running applications. 

When you go to the task bar and select DAISY Pipeline, it says that DAISY Pipeline is running. If you press enter on it, it does not move to the foreground. One must go to the task bar, select DAISY Pipeline and then use the context menu  to bring up the selections, and then you can get to settings or create a job. This will move the DAISY Pipeline to the foreground and put it back into the ALT + Tab order.

I am not sure about this, but I normally use the NVDA key + B to go to the task bar. I am finding it difficult to get to the task bar after launching the DAISY Pipeline. I don't know where this behavior is coming from. It could be my configuration and I am using a beta of NVDA.

I am using:
DAISY Pipeline releases/tag/0.2.0-alpha
Windows 11 2022 H2
NVDA  Version: 2022.4rc1 (2022.4.0.27372)

**Comment by GeorgeKerscher** [comment]
Windows key + B is taking me to the task bar properly. I made a mistake in the original comment saying  NVDA key + B. So, that is OK, and the rest of the unusual behavior is still present.

**Comment by NPavie** [comment]
Hi Richard, thank you for the feedback.

I'll change things a bit on this side of the app execution model for the next release : 
the behavior I often see in similar applications on Windows is an information dialog stating the app keeps running in the tray, sometimes also asking if the user wants to keep the app running on closing the main window.
In the last case, users could then choose the default behavior of the window closing action with this dialog, and change it back in the app settings if wanted.

**Comment by bertfrees** [comment]
On macOS there is "Close window" and "Quit". I'm not sure if "Quit" should be called "Exit" on Windows, and whether it should be in the "File" menu. It seems inconsistent, but maybe it is consistent with other Windows applications. Anyway, it is our intention to keep the tray active and the engine running in the background when the window is closed with the close button. It might be somewhat unexpected for some users, but I would like to keep this behavior.

What we could do is warn the user that the engine is not stopped, and explain how to stop it, or ask if he would also like to stop the engine or not. In that case there should also be a "Don't ask me again" button. The benefit of this approach is that it is completely transparent, the downside is that some users are not interested in such details.

**Comment by bertfrees** [comment]
Thanks Nico, yes a configuration setting also makes sense.

**Comment by ways2read** [comment]
Perhaps when the app is started the UI window should be shown even if the engine is already running. That would address the user confusion I was concerned about.

**Comment by bertfrees** [comment]
Agree.
